addons/mail/static/src/new/discuss/discuss.js

```diff
@@ -165,12 +165,16 @@ export class Discuss extends Component {
 
     async renameGuest({ value: name }) {
         const newName = name.trim();
-        if (this.store.guest?.name !== newName) {
+        if (this.store.guest && this.store.guest.name !== newName) {
```

Comment:
It is the same I think.

Reply:
Not really, if `newName` undefined and no store guest, it passes

Reply:
But it assumes we allow renaming of user, which is not desirable here